code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -883,8 +883,8 @@
 	force = 20
 	mag_type = /obj/item/ammo_box/magazine/m556/rifle/small
 	init_mag_type = /obj/item/ammo_box/magazine/m556/rifle/small
-	extra_damage = 30
-	extra_penetration = 0.1
+	extra_damage = 33
```

Comment:
The Colt Rangemaster was originally a 7.62 ammo gun, so I thought 30% was alright since it's a 10-round semi auto. But yeah, I'll bring it down to 20% for good measure.